api/src/db/init.ts

```diff
@@ -13,15 +14,24 @@ async function initDb() {
 
     // Syncs the model
     try {
-        await sequelize.sync({force: true});
+        await sequelize.sync({ force: true });
         console.log('Database sync sucessful');
     } catch (error) {
         console.log(`Error while trying to sync the model with the database : ${error}`);
     }
 
     // Inserts an article, to replace with fake data.
     try {
-        await Article.create({title: 'Title', description: 'This is a test', content: 'bla bla bla'});
+        await Article.create({ title: faker.random.word(), description: faker.lorem.sentence(), content: faker.lorem.sentences() });
```

Comment:
Well, you already insert 10 other fake data below, you don't need this one anymore